Zeplin.sketchplugin/Contents/Sketch/utils.cocoascript

```diff
@@ -178,6 +178,14 @@ var getShareId = function (context) {
     return nil;
 }
 
+var getSourceFileName = function (context) {
+    try {
+        return context.document.displayName();
+    } catch (error) {
+        log("Getting source file name failed with error “" + error + "”."));
+    }
```

Comment:
To keep things consistent we should return `nil` at the end of the function

Reply:
✅